src/app/code/community/Omise/Gateway/Block/Form/Offsiteinstallment.php

```diff
@@ -48,8 +48,7 @@ public function isInterestFree()
     public function getMonthlyPaymentAmount($saleAmount, $monthCount, $interestRatePC)
     {
         $interestRate = $this->isInterestFree() ? 0 : $interestRatePC/100;
-        $VAT = 0.07; // Should this be in config, and also country dependent??
```

Comment:
@jonrandy 🤣

Reply:
Bahahahahahahahaha.... See, I even level the same criticisms at myself! I knew there was a reason it was bothering me! 🤣 🙄

Reply:
no problem.. I am just happy that in the end, we have the same idea. The difference only is that in Mag2 responsible for that is JS